ext/filter/filter.stub.php

```diff
@@ -4,17 +4,13 @@
 
 function filter_has_var(int $type, string $variable_name): bool {}
 
-/** @param array|int $options */
-function filter_input(int $type, string $variable_name, int $filter = FILTER_DEFAULT, $options = null): mixed {}
+function filter_input(int $type, string $variable_name, int $filter = FILTER_DEFAULT, array|int|null $options = null): mixed {}
 
-/** @param array|int $options */
-function filter_var(mixed $variable, int $filter = FILTER_DEFAULT, $options = null): mixed {}
+function filter_var(mixed $variable, int $filter = FILTER_DEFAULT, array|int|null $options = null): mixed {}
 
-/** @param array|int $options */
-function filter_input_array(int $type, $options = FILTER_DEFAULT, bool $add_empty = true): array|false|null {}
+function filter_input_array(int $type, array|int|null $options = null, bool $add_empty = true): array|false|null {}
```

Comment:
The default value was changed from `FILTER_DEFAULT` to `null` to be consistent with the previous `$options` parameters, but I'm not sure what to do exactly. Technically, we could declare both as far as I see.